src/modelfree/score_agent.py

```diff
@@ -50,6 +57,62 @@ def get_empirical_score(_run, env, agents, episodes, render=False):
     return result
 
 
+def _clean_video_directory_structure(observer_obj):
+    """
+    A simple utility method to take saved videos within a Sacred run structure and clean
+    up the file pathways, so that all videos are organized under a "videos" directory
+
+    :param observer_obj: A Sacred FileStorageObserver object
+    :return: None
+    """
+    basedir = observer_obj.dir
+    video_files = glob.glob("{}/*.mp4".format(basedir))
+    metadata_files = glob.glob("{}/*metadata.json".format(basedir))
+    if len(video_files) == 0:
+        return
+
+    new_video_dir = os.path.join(basedir, "videos")
+    os.mkdir(new_video_dir)
+    new_video_metadata_dir = os.path.join(new_video_dir, "metadata")
+    os.mkdir(new_video_metadata_dir)
+    for video_file in video_files:
+        base_file_name = os.path.basename(video_file)
+        os.rename(video_file, os.path.join(new_video_dir, base_file_name))
+
+    for metadata_file in metadata_files:
+        base_file_name = os.path.basename(metadata_file)
+        os.rename(metadata_file, os.path.join(new_video_metadata_dir, base_file_name))
+
+
+def _save_video_or_metadata(env_dir, saved_video_path):
+    """
+    A helper method to pull the logic for pattern matching certain kinds of video and metadata
+    files and storing them as sacred artifacts with clearer names
+
+    :param env_dir: The path to a per-environment folder where videos are stored
+    :param saved_video_path: The video file to be reformatted and saved as a sacred artifact
+    :return: None
+    """
+    env_number = env_dir.split("/")[-1]
+    video_ptn = re.compile(r'video.(\d*).mp4')
+    metadata_ptn = re.compile(r'video.(\d*).meta.json')
+    video_search_result = video_ptn.match(saved_video_path)
+    metadata_search_result = metadata_ptn.match(saved_video_path)
+
+    if video_search_result is not None:
+        episode_id = video_search_result.groups()[0]
+        sacred_name = "env_{}_episode_{}_recording.mp4".format(env_number, int(episode_id))
+
+    elif metadata_search_result is not None:
+        episode_id = metadata_search_result.groups()[0]
+        sacred_name = "env_{}_episode_{}_metadata.json".format(env_number, int(episode_id))
+
```

Comment:
delete whitespace

Reply:
I'll make the change for the sake of getting the branch merged rather than going back and forth on it, but I actually think that having whitespace in cases like this is better/makes the code more readable than the alternative